serverless/client/client.go

```diff
@@ -145,21 +146,21 @@ func (pb *ProofBuilder) InclusionProof(ctx context.Context, index uint64) ([][]b
 // This function uses the passed-in function to retrieve tiles containing any log tree
 // nodes necessary to build the proof.
 func (pb *ProofBuilder) ConsistencyProof(ctx context.Context, smaller, larger uint64) ([][]byte, error) {
-	nodes, err := merkle.CalcConsistencyProofNodeAddresses(smaller, larger)
+	nodes, err := proof.Consistency(smaller, larger)
 	if err != nil {
 		return nil, fmt.Errorf("failed to calculate consistency proof node list: %w", err)
 	}
 
 	hashes := make([][]byte, 0)
```

Comment:
retrospective nit: Looks like the code below this point is identical to `InclusionProof`. Consider sharing (not in this PR).